### PR TITLE
Stop spamming handle service logs

### PR DESCRIPTION
**Pull request by MrCreosote**
Now there's a status method, can just use that

Also move the check method into the server class. No point in putting it
in the methods class.

### CODE REVIEW

File: src/us/kbase/workspace/kbase/WorkspaceServerMethods.java [hunk 1]
```diff
@@ -96,45 +92,24 @@
...
 		this.idFacBuilder = idFacBuilder;
-		this.handleServiceUrl = handleServiceUrl;
 		this.auth = auth;
 	}
 	
 	public ConfigurableAuthService getAuth() {
 		return auth;
 	}
 	
-	public URL getHandleServiceURL() {
```
Comment (scanon) [inline src/us/kbase/workspace/kbase/WorkspaceServerMethods.java]: Why was this removed?  Where is that call now?

Comment (scanon) [reply src/us/kbase/workspace/kbase/WorkspaceServerMethods.java]: Maybe I get it... There is still a method in the config part.

Comment (MrCreosote) [reply src/us/kbase/workspace/kbase/WorkspaceServerMethods.java]: the handle service stuff living in that class was cruft from older implementations - there was no real reason for it to be there, and it made more sense just to move all that code to the main server class.

Comment (MrCreosote) [reply src/us/kbase/workspace/kbase/WorkspaceServerMethods.java]: And yeah, the main server class is now getting a client from the object returned by `initWorkspaceService` or whatever it's called, rather than pulling the url from this class.
